File: CodeBattle.java/src/main/java/clientlib/domain/server/PlanetInfo.java
package clientlib.domain.server;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;


@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PlanetInfo {
    private long id;
    private long droids;
    private String owner;
    private PlanetType type;
    private List<Long> neighbours;
}

File: CodeBattle.net/CodeBattleNetLibrary/PlanetInfo.cs
using Newtonsoft.Json;

namespace CodeBattleNetLibrary
{
	public class PlanetInfo
	{
		[JsonProperty("id")] public int Id { get; set; }
		[JsonProperty("droids")] public int Droids { get; set; }
		[JsonProperty("owner")] public string Owner { get; set; }
		[JsonProperty("type")] public string Type { get; set; }
		[JsonProperty("neighbours")] public int[] Neighbours { get; set; }
	}
}
